src/client/telemetry/constants.ts

```diff
@@ -21,6 +21,9 @@ export enum EventName {
     REFACTOR_EXTRACT_VAR = 'REFACTOR_EXTRACT_VAR',
     REFACTOR_EXTRACT_FUNCTION = 'REFACTOR_EXTRACT_FUNCTION',
     REPL = 'REPL',
+    SELECT_INTERPRETER = 'SELECT_INTERPRETER',
+    ENTER_INTERPRETER_PATH_BUTTON = 'ENTER_INTERPRETER_PATH_BUTTON',
+    INPUT_INTERPRETER_PATH_CHOICES = 'INPUT_INTERPRETER_PATH_CHOICES',
```

Comment:
It might be better to group these via the name, to make the relationship more clear:

```suggestion
    SELECT_INTERPRETER = 'SELECT_INTERPRETER',
    SELECT_INTERPRETER_ENTER_BUTTON = 'SELECT_INTERPRETER_ENTER_BUTTON',
    SELECT_INTERPRETER_ENTER_CHOICES = 'SELECT_INTERPRETER_ENTER_CHOICES',
```

Furthermore, "choices" implies multiple selections even though there was only one.  So:

```suggestion
    SELECT_INTERPRETER = 'SELECT_INTERPRETER',
    SELECT_INTERPRETER_ENTER_BUTTON = 'SELECT_INTERPRETER_ENTER_BUTTON',
    SELECT_INTERPRETER_ENTER_CHOICE = 'SELECT_INTERPRETER_ENTER_CHOICE',
```